README.md

```diff
@@ -675,6 +675,22 @@ $ lerna exec --scope my-component -- ls -la
 $ lerna run --scope toolbar-* test
 ```
 
+#### --since [ref]
```

Comment:
I moved this to just after the `--scope` arg since it relates very closely to it.